traffic_portal/app/src/app.js

```diff
@@ -457,7 +457,6 @@ var trafficPortal = angular.module('trafficPortal', [
 
         .run(function($log, applicationService) {
             $log.debug("Application run...");
-            applicationService.startup();
```

Comment:
wasn't this important?

Reply:
no. i thought it might be useful but i never used it. there's an init function in applicationService you can use instead if you need to do something at startup.

Reply:
:+1: that sounds like a more proper use of Angular anyway.